docs/install/Knative-with-AKS.md

```diff
@@ -195,12 +195,12 @@ your Knative installation, see
    completed until the upgrade process finishes.
```

Comment:
@ZhiminXiang here are the changes that I think should happen in the host specific install files

Reply:
OK. I will copy them into other host specific files.

Reply:
Done

Reply:
looks good, thanks!